robot_log_visualizer/ui/gui.py

```diff
@@ -669,6 +689,158 @@ def dragEnterEvent(self, event):
         else:
             event.ignore()
 
+    def variableTreeWidget_on_right_click(self, item_position):
+        # check if the variable tree widget is empty
+        if self.ui.variableTreeWidget.topLevelItemCount() == 0:
+            menu = QtWidgets.QMenu()
+            menu.addAction("Open a mat file")
+            action = menu.exec_(self.ui.variableTreeWidget.mapToGlobal(item_position))
+            if action is None:
+                return
+            if action.text() == "Open a mat file":
+                self.open_mat_file()
+
+            return
+
+        # find the item from the position
+        item = self.ui.variableTreeWidget.itemAt(item_position)
+        if item is None:
+            return
+
+        if item.childCount() == 0:
+            return
+
+        # the child has to be a leaf
+        if item.child(0).childCount() != 0:
+            return
+
+        # check the number of children
+        item_size = item.childCount()
+        item_path = self.get_item_path(item)
+        item_key = "/".join(item_path)
+
+        menu = QtWidgets.QMenu()
+
+        add_3d_point_str = "Show as a 3D point"
+        remove_3d_point_str = "Remove the 3D point"
+        use_as_base_position_str = "Use as base position"
+        use_as_base_orientation_str = "Use as base orientation"
```

Comment:
As mentioned F2F, it would be useful to differentiate the label in 
```
Use as base orientation (rpy)
```
and
```
Use as base orientation (xyzw quaternion)
```
depending on the dimension of the specified data

Reply:
Done in https://github.com/ami-iit/robot-log-visualizer/pull/74/commits/8689b378f38e0bd771e0970c729a8aa29b2509ff